prediction/src/algorithms/segment/src/visualization.py

```diff
@@ -0,0 +1,110 @@
+"""This file contains methods that help to visualize the training cubes of annotations"""
+
+import matplotlib.pyplot as plt
+from matplotlib.widgets import Slider
+
+
+def cuboid_show_slider(cuboid, axis=2, is_mask=False, **kwargs):
+    """Display a 3D ndarray with a slider to move along the third dimension.
+
+    Extra keyword arguments are passed to imshow
+    (Kudos to http://nbarbey.github.io/2011/07/08/matplotlib-slider.html)
+
+    Args:
+        cuboid: three-dimensional array that should be visualized
+        axis: index of the axis that should be changeable using the slider
+        is_mask: whether the cuboid that should be plotting is a binary mask
+    """
+    import matplotlib.pyplot as plt
```

Comment:
How come the inline imports? (please explain in a comment, etc.)

Reply:
It's unnecessary, thanks ;)

Reply:
Done in a1d18abdf4bacf6665ede1048e6d4652ba09f7d9